src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] (Optional) Kindly import `createStore` instead of `configureStore`. Remember that for this project, you are to import `createStore` from `'redux'` and not `configureStore.` Kindly implement this important change. 👍

Reply:
hi dear @Gambit142 I tried to use the `createStore` from `redux` but this happens
![Screenshot from 2022-05-02 17-13-27](https://user-images.githubusercontent.com/96317674/166335903-dadc7a11-76d3-443f-9f2c-0268fc98888c.png)

Reply:
Hi @Juanse7793 , you can try this syntax `import { legacy_createStore as createStore } from 'redux'`